indico/modules/events/abstracts/forms.py

```diff
@@ -556,6 +559,31 @@ class SendNotificationsMixin(object):
     send_notifications = BooleanField(_("Send email notifications"), default=True)
 
 
+class InvitedAbstractMixin(IndicoForm):
+    users_with_no_account = IndicoRadioField(_('Type of user'), [DataRequired()], default='existing',
+                                             choices=(('existing', _('Indico user')),
+                                                      ('new', _('New Indico user'))))
+    submitter = PrincipalField(_('Submitter'),
+                               [HiddenUnless('users_with_no_account', 'existing'), DataRequired()], allow_external=True,
+                               description=_('The person invited to submit the abstract'))
+    first_name = StringField(_('First name'), [HiddenUnless('users_with_no_account', 'new'), DataRequired()])
+    last_name = StringField(_('Family name'), [HiddenUnless('users_with_no_account', 'new'), DataRequired()])
+    email = EmailField(_('Email address'), [HiddenUnless('users_with_no_account', 'new'), DataRequired()],
+                       filters=[lambda x: x.lower() if x else x])
+
+    def __init__(self, *args, **kwargs):
+        self.event = kwargs['event']
+        super(InvitedAbstractMixin, self).__init__(*args, **kwargs)
+
+    def validate(self):
+        from indico.modules.events.abstracts.util import can_create_invited_abstracts
+        if not can_create_invited_abstracts(self.event):
+            raise ValidationError(_('You have to create an "Invited" abstract notification template in order to '
```

Comment:
I think showing it here is a bit late. I'd rather show the message above the form before you even try to submit it (and might have filled everything out just to realize you can't submit it..)

Reply:
OK nevermind, I saw below that we disable the link to invite. Not sure if we need to bother with a translatable error here... You can't really reach this code except when you manually load the form to submit an invited abstract.. and you could just 403 there if it's not properly configured